api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -50,6 +50,15 @@ default MigrateTable dropBackup() {
     throw new UnsupportedOperationException("Dropping a backup is not supported");
   }
 
+  /**
+   * @param numReaders the number of concurrent file read operations to use per partition
+   * @return this for method chaining
+   */
+  default MigrateTable withParallelReads(int numReaders) {
+    throw new UnsupportedOperationException(
```

Comment:
I think we can default to 1?

Reply:
> I think we can default to 1?

/LGTM,  But I'm not familiar with JAVA,  Could you give more detail about how set this default to 1 , many thanks. @jackye1995

Reply:
I think unsupported is right here, since we are basically saying that the method is unsupported not that there isn't a default.

Reply:
Also I think our default should probably be > 1

Reply:
Thanks @RussellSpitzer for your reviews again, So I just Keep it as it is using unsupported.